zds/stats/management/commands/parse_logs.py

```diff
@@ -93,57 +93,50 @@ def is_treatable(self, dict_result):
 
         return False
 
-    def flush_denormalize(self, class_name, field_in_class, field_in_log, list_of_datas):
-        module = 'zds.stats.models'
-        obj = __import__(module, globals(), locals(), [class_name])
-        cls = getattr(obj, class_name)
-        data_existants = cls.objects.values_list(field_in_class, flat=True)
-        data_for_save = list(set(list_of_datas) - set(data_existants))
-
-        for my_data in data_for_save:
-            create_arg = {field_in_class: my_data}
-            cls.objects.create(**create_arg)
-
     @transaction.atomic
     def flush_data_in_database(self):
-        my_sources = []
-        my_os = []
-        my_browsers = []
-        my_devices = []
-        my_cities = []
-        my_countries = []
         new_logs = []
         keys = ['id_zds', 'content_type', 'remote_addr', 'hash_code', 'body_bytes_sent', 'timestamp',
-                'dns_referal', 'os_family', 'os_version', 'browser_family', 'browser_version',
-                'device_family', 'request_time', 'country', 'city']
+                'os_version', 'browser_version', 'request_time']
         for data in self.datas:
```

Comment:
data et chunk...

vl'a le débat.


Le 26/08/2016 à 17:49, victor felder a écrit :
>
> In zds/stats/management/commands/parse_logs.py 
> <https://github.com/zestedesavoir/zds-site/pull/3785#discussion_r76441352>:
>
> >          new_logs = []
> >          keys = ['id_zds', 'content_type', 'remote_addr', 'hash_code', 'body_bytes_sent', 'timestamp',
> > -                'dns_referal', 'os_family', 'os_version', 'browser_family', 'browser_version',
> > -                'device_family', 'request_time', 'country', 'city']
> > +                'os_version', 'browser_version', 'request_time']
> >          for data in self.datas:
>
> Tu peux aussi utiliser le singulier et le pluriel, à savoir |datum| et 
> |data|. Ou |item| et |data|.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3785/files/94bd5c45f0c9893d42bb626ae7616c2a68abef21..fc4b9c0a275056cb2e01499dfade3abc700ab0cc#r76441352>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABc_xTliXr7JolizmBBcyFuko5fF9PkAks5qjwrsgaJpZM4JqNke>.
>